index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
``` To address the performance issue I mentioned, I recommend refactoring the queries to use better indexing and query optimization techniques.